src/cli.ts

```diff
@@ -55,12 +60,32 @@ export async function run(): Promise<void> {
 
   if (cliOptions.help) {
     printUsage(argsWithHelp);
+    return;
   }
-  else if (cliOptions.version) {
+  if (cliOptions.version) {
     console.log(getVersion());
+    return;
   }
-  else {
-    await startServer(options);
+
+  const serverInfos = await startServers(options);
+
+  if (serverInfos.length === 1) {
+    const urls = getServerUrls(options, serverInfos[0]!.server);
+    console.log(`Files in this directory are available under the following URLs
+    applications: ${
+                  url.format(urls.serverUrl)}
+    reusable components: ${url.format(urls.componentUrl)}
+  `)
+  } else {
+    const dispatchServer = serverInfos.find(s => s.kind === 'dispatch');
```

Comment:
Rather than return a simple array of servers, would it be nicer to return a struct that contains a reference to the dispatch server, app server and array of component servers? Are there any top-level properties / methods that would make sense?